Problem2.py

```diff
@@ -0,0 +1,39 @@
+'''
+Problem: Merge Sorted Arrays
+Time Complexity: O(n), where n is given elements
```

Comment:
Kindly check TC. You have two arrays here and the size of the two arrays is different on which you are iterating.